lib/components/getUsersForGroup.js

```diff
@@ -71,7 +71,7 @@ GroupUsersFinder.prototype.forEachIter = function forEachIter (member, acb) {
     // prefer to use member.dn instead member.cn, because cn may not be unique
     ad.getUsersForGroup(this.opts, member.dn, (err, nestedUsers) => {
       if (err) throw err
-      nestedUsers.forEach(u => this.users.set(u.dn, u))
```

Comment:
This was undefined when I debugged it. But only when there is a group  which has at least 2 other groups and these groups have at least 1 member. `u.distinguishedName` was available though and changing it fixed all my problems.